setup.cfg

```
@@ -0,0 +1,68 @@
+[metadata]
+name = cobbler_tftp
+author = Team Cobbler
+author_email = cobbler.project@gmail.com
+url = https://github.com/cobbler/cobbler-tftp
+description = The tftp server daemon for Cobbler
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords = cobbler, tftp
+license = GPL v2.0 License
+classifiers =
+  Natural Language :: English
+  Operating System :: OS Independent
+  Programming Language :: Python :: 3
+
+[options]
+zip_safe = True
+package_dir=
+  =src
+packages = find:
+include_package_data = True
+setup_requires =
+  setuptools<=44.11.0
```

So I believe that we don't need something that is smaller or equal to 44.11.0. I believe that atm we should be able to install this project also with more modern versions of `setuptools`. So I believe we want this if I am not mistaken:

```suggestion
  setuptools>=44.11.0
```